docs/hosting/environment-variables/environment-variables.md

```diff
@@ -146,7 +146,7 @@ Refer to [User management](/hosting/authentication/user-management-self-hosted/)
 
 | Variable | Type  | Default  | Description |
 | :------- | :---- | :------- | :---------- |
-| `EXECUTIONS_PROCESS` (**deprecated**) | Enum string: `main`, `own` | `own` | **Deprecated**. Whether n8n executions run in their own process or the main process. <br><br>Refer to [Execution modes and processes](/hosting/scaling/execution-modes-processes/) for more details. |
+| `EXECUTIONS_PROCESS` (**deprecated**) | Enum string: `main`, `own` | `main` | **Deprecated**. Whether n8n executions run in their own process or the main process. <br><br>Refer to [Execution modes and processes](/hosting/scaling/execution-modes-processes/) for more details. |
```

Comment:
Looks like [this URL](https://docs.n8n.io/hosting/scaling/execution-modes-processes/#own) should be updated as well?

<img width="718" alt="image" src="https://github.com/n8n-io/n8n-docs/assets/44588767/85c3f1c2-18b7-4393-8d7a-fa256ef3281f">

Did not check if there are other places that might also need updating.